motion_planning_config:
  parameters:
    name: "Planning pipeline"
    runs: 50                      # Number of runs per scene, interface, collision detector and each planning algorithm
    timeout: 10                   # Default 10.0
  interface:                      # [PlanningPipeline, MoveGroupInterface]
    - MoveGroupInterface
    - PlanningPipeline
  collision_detector:             # [FCL, Bullet]
    - FCL
    - Bullet                      # Not configurable when using the MoveGroupInterface  
  planning_pipelines:  
    - name: ompl                                                              # REQUIRED
      planners:                                                               # REQUIRED
        - RRTConnectkConfigDefault
    - name: chomp
      planners:
        - CHOMP 

